configuration/config.go

```diff
@@ -0,0 +1,53 @@
+package configuration
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/spf13/viper"
+)
+
+const LoadConfigErrMsg = "Error loading config file"
+
+type Auth struct {
+	user     string
+	password string
+	host     string
+	port     string
+	apitkn   string
+	URL      string
+}
+
+// LoadConfig from Viper
+func LoadConfig() {
+	viper.SetConfigName("config")
+	viper.AddConfigPath("configuration")
+	if err := viper.ReadInConfig(); err != nil {
+		log.Fatalf("%s: %s", LoadConfigErrMsg, err)
+	}
+}
+
+func ApiAuth() *Auth {
+	return &Auth{
+		apitkn: viper.GetString("soccerama.apitkn"),
+	}
+}
+
+func (a *Auth) GetAPI() {
+	a.URL = fmt.Sprintf("https://api.soccerama.pro/v1.2/competitions?api_token=%s&include=country",
+		a.apitkn)
+}
```

Comment:
We gonna end up with multiple endpoints like `competitions`, `seasons`, `players`, and so on. For that I would slipt the this url in few components.

For instance:

```
type URL struct {
    addr, version, token, endpoint, params string
}

func (u *URL)getURL() string {
return fmt.Sprintf("%s....", u.addr, u.version,...)
}
```

and initialize it as:
```
endpoint := "https://api.soccerama.pro/"
version := "v1.2"

url := &URL{
endpoint, version, apiToken(), competitionEndpoint(), competitionParams()
}

url := getURL()
```

Something like this...

This way we can keep a list of the available endpoints and a list of the necessary params that each endpoint may use.